student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom, name = 'Unknown')
+    super(name)
```

Comment:
- [ ] Please make sure to pass the right parameters to the parent constructor like the following:
```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
```